src/PJ_sch.c

```diff
@@ -0,0 +1,206 @@
+/*
+** Author : Piyush Agram
+** This code is in the public domain and implements the SCH coordinate system first developed at JPL.
```

Comment:
I believe the contributor is a U.S. federal employee, and therefore cannot define a license other than public domain for contributions like this (or at least not easily without some bureaucratic mountain moving). I think there are many situations (see GEOTRANS) where code is contributed in this fashion.